scripts/settings_list.py

```diff
@@ -0,0 +1,53 @@
+import yaml
+import pprint
+
+class SettingsList():
+  list_of_dicts= list();
+
+  def sl_print(self):
```

Comment:
Agree.  Instead of using print I used the python internal __str__ method to print the representation.